src/lib/core/datetime.h

```diff
@@ -72,18 +101,22 @@ tnt_datetime_to_string(const struct datetime *date, char *buf, ssize_t len);
  * @retval length of a resultant text
  */
 size_t
-tnt_datetime_strftime(const struct datetime *date, char *buf, size_t len,
+datetime_strftime(const struct datetime *date, char *buf, size_t len,
 		      const char *fmt);
 
 void
-tnt_datetime_now(struct datetime *now);
+datetime_now(struct datetime *now);
 
 /**
  * Convert @sa datetime to @sa dt_tm
  */
 void
 datetime_to_tm(const struct datetime *date, struct tnt_tm *tm);
 
+#define STRONG_SYMBOL_ALIAS(name, alias) _STRONG_SYMBOL_ALIAS(name, alias)
```

Comment:
Apparently, this is not working on darwin, they refuse to handle `__attribute__(alias)` or `weak,alias`. But there is harder way workaround:

```diff
commit 5fb580b7fc11ef76593052a5554cb6e52179585c
Author: Timur Safin <tsafin@tarantool.org>
Date:   Wed Dec 15 15:46:15 2021 +0300

    temp - fix macosx

diff --git a/src/lib/core/datetime.h b/src/lib/core/datetime.h
index 05223d8a8..9c71eb2ca 100644
--- a/src/lib/core/datetime.h
+++ b/src/lib/core/datetime.h
@@ -114,8 +114,15 @@ void
 datetime_to_tm(const struct datetime *date, struct tnt_tm *tm);
 
 #define STRONG_SYMBOL_ALIAS(name, alias) _STRONG_SYMBOL_ALIAS(name, alias)
+#ifdef __APPLE__
+#define _STRONG_SYMBOL_ALIAS(name, aliasname) \
+	__asm__(".globl _" #aliasname); \
+	__asm__(".set _" #aliasname ", _" #name); \
+	extern __typeof(name) aliasname;
+#else
 #define _STRONG_SYMBOL_ALIAS(name, aliasname) \
 	extern __typeof(name) aliasname __attribute__((alias (#name)));
+#endif
 
 #if defined(__cplusplus)
 } /* extern "C" */
```